miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/ExperimentService.java

```diff
@@ -2,18 +2,24 @@
 
 import java.io.IOException;
 import java.util.Collection;
+import java.util.List;
 import java.util.Map;
 
 import uk.ac.bbsrc.tgac.miso.core.data.Experiment;
+import uk.ac.bbsrc.tgac.miso.service.security.AuthorizationException;
 
 public interface ExperimentService {
-  public long save(Experiment experiment) throws IOException;
+  public void delete(Experiment experiment) throws IOException;
 
   public Experiment get(long experimentId) throws IOException;
 
+  public Map<String, Integer> getColumnSizes() throws IOException;
+
   public Collection<Experiment> listAll() throws IOException;
 
-  public Collection<Experiment> listAllWithLimit(long limit) throws IOException;
+  public Collection<Experiment> listAllByLibraryId(long id) throws AuthorizationException, IOException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'uk.ac.bbsrc.tgac.miso.service.security.AuthorizationException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)